core/test/qa_Settings.cpp

```diff
@@ -71,13 +76,15 @@ struct Source : public Block<Source<T>> {
     gr::Size_t n_samples_max      = 1024;
     float      sample_rate        = 1000.0f;
 
-    void settingsChanged(const property_map& /*oldSettings*/, property_map& newSettings, property_map& fwdSettings) {
+    void
+    settingsChanged(const property_map & /*oldSettings*/, property_map &newSettings, property_map &fwdSettings) {
         // optional init function that is called after construction and whenever settings change
         newSettings.insert_or_assign("n_samples_max", n_samples_max);
         fwdSettings.insert_or_assign("n_samples_max", n_samples_max);
     }
 
-    [[nodiscard]] constexpr T processOne() noexcept {
+    [[nodiscard]] constexpr T
+    processOne() noexcept {
```

Comment:
It looks like you are using wrong clang-format. The return type must be on the same line. This appear in several places. Please check.